_site/index.html

```diff
@@ -96,24 +99,12 @@ <h2 class="title__heading">Why Open Source?</h2>
 
     <section class="section section-upcoming">
       <div class="section__content">
-        <h2 class="section__header">Upcoming Events</h2>
+        <h2 class="section__header">Current Events</h2>
 
-        <!-- <p class="title__description">
-          Follow us on
-          <a href="https://instagram.com/cbitosc" target="_blank" rel="noopener">Instagram</a>,
-          <a href="https://twitter.com/cbitosc" target="_blank" rel="noopener">Twitter</a>, and
-          <a href="https://facebook.com/cbitosc" target="_blank" rel="noopener">Facebook</a>
-          for updates on upcoming events.
-        </p> -->
-        <center><a href="/img/COSC Internship.pdf" download><img class="upevent-img" src="/img/intern-img.jpg" style=""></a></center>
-        <center><a class="reg-btn"  href="https://forms.gle/xPYgjzR89EbnLUez7">Register</a></center>
-        <!-- <div class="cards">
-          <div class="event card">
-            <img class="card__image card__image--special_1" src="/img/events/websession-2020.jpg" loading="lazy">
-            <div class="card__info">
-              <p>Session on Web Development</br>14 March 2020</br><a href="https://forms.gle/2Axo5Fx9PPSPCAcj7">click here for registration</a>
-          </div>
-        </div> -->
+        
+        <center><a href="/img/COSC Internship.pdf" download><img class="upevent-img" src="/img/intern-img.jpg"></a></center>
```

Comment:
If the pdf can be viewed in browser then let it be. Download force "download first" option, it's just a usability issue.